### DISCUSSION TITLE
Option to Hide Account Country

### DISCUSSION THREAD

**Discussion by LucyKosaki** [ppy/osu#28120] [Ideas] [answered]
I would like to suggest the option to hide your country (which will except you from rankings).

Some people are uncomfortable showing their countries online for different privacy-related reasons.

On at least 2 different occasions I already had the situations where Vtubers had to refuse playing Osu with viewers on stream
because they did not want the game to dox their country location.

Both at first accidently made an account, noticed that it displays their country, then were forced to do something about it.
The first let their account get deleted through email, the other one paid for premium to be able to let their name changed, only for it to still being shown as former name.

I fell for that trap myself too, paying for premium to change my name in hopes of my nick not getting associated with the displayed country, but since I am from the EU I could at least request deletion by EU law.

There are also more streamers, who are forced to play the game offline because of this.

I don't know any other game, where it shows your country with no option to hide it, I think that right now it's kind of a privacy issue in general.
You don't expect the game to country-dox you like that, you cant delete your account unless contacting support or you are forced to pay just to not have your nick associated with a country, only for it to be shown anyways unless you are from EU.

I think this is a big issue.

Most people are playing casually and not for tournaments and rankings anyways and aside from those the country is not needed to be displayed.

Please add an option to hide or remove your country.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu-web/issues/8033

**Reply by SolsticeSpectrum** [reply]
Could there be some public poll on this? Maybe on Discord? Peppy said that it would only be considered if large percentage of users wants this. But in order to find out if they do, there should be some poll that more people can notice.

**Reply by sinc0s** [reply]
The idea of a way to hide your country flag has been brought up many times at this point:

'Feature to make your Country private' (what you linked): https://github.com/ppy/osu-web/issues/8033
'Streaming mode needed' (4th most upvoted discussion): https://github.com/ppy/osu/discussions/27074
'Adding "Streamer Mode"': https://github.com/ppy/osu/issues/12547
'Hope to provide the function of turning off the display of flags.': https://github.com/ppy/osu/discussions/24369
'Add display flag Icon option': https://github.com/ppy/osu/discussions/20154
'Add an option to toggle country/region flag display': https://github.com/ppy/osu/discussions/25956
'Ability to make country display private': https://github.com/ppy/osu/discussions/24560
'A "Stream mode" with the feature to hide the country flag and something other': https://github.com/ppy/osu/discussions/22089

In the post you linked, ppy said: "Won't consider unless there's an actual large percentage of users wanting this". Considering the amount of posts and discussion asking for the feature to be implemented, and the unanimous support for it's implementation, there is at least a substantial amount of people wanting this feature to exist. There are many reasons a person would use this that have already been talked about. Simply linking past responses every time this feature is requested is negligent.

**Reply by bdach** [reply]
@sinc0s "streamer mode" is a different feature request than "allow user to opt out of country tagging everywhere" which is what i understand the op wants.

**Reply by sinc0s** [reply]
The request for a streamer mode has always come with the request to be able to hide your country flag. That is why I linked them as well. I simply aim to show that the feature **is** heavily requested and has been multiple times for a long while, not to mention how important it is to some people (China restrictions, VTubers, etc.).

**Reply by peppy** [reply]
If it was added, it would be a client-side setting (aka streamer mode). Removing country from user profiles is not something we are considering for osu!. You are welcome to use a VPN if it's that important to you.

**Reply by sinc0s** [reply]
This is fair I believe. From what I have read the main concerns are either showing some specific flags (China restrictions) or with general privacy (not wanting to show the country you're from). I think most people are asking for the client-side setting you mentioned, which is also what I was talking about before, sorry if that was unclear.

**Reply by LucyKosaki** [reply]
@sinc0s A client-side solution would not solve any of the privacy and doxing issues, yeah. If the game shows your nick on stream, anyone can just look it up on the website. And if we have to log out in order to not show our account name, then there would be no difference to the current situation, where you can't play the game on stream properly.

If adding a straight solution for this is not something, that peppy wants to do, I think at least it would be important to make it very clear, that you are going to dox your country when you create an account AND there should be an easier way (for non-EU players too), to change and delete previous names or delete accounts fast before things get out of hand (for example if you accidently showed your account on stream unaware, that people are able to see on the website). Because right now at least 3 times in my circle alone it happened, that people felt trapped or forced to pay just to be able to protect themselves.

**Reply by Judge1st** [reply]
There's also another side of it, having flag hiding option is a must if you have such a strict profile flag policy, some people can't change their flag because the conditions you made are just so insanely strict (and before flag change update it was not even possible), if you're not letting people change their profile flag easily you should give them ability to hide it.

osu! showing your flag almost in every place where it's possible, and in case when you have a wrong flag that you don't want to be associated with at the very least you would be satisfied by hiding it, it's very bothering every second it happens.
